src/components/Navbar.vue

```diff
@@ -91,6 +95,12 @@
             Link To Us
           </b-dropdown-item>
           -->
+          <b-dropdown-item
+            href="https://medium.com/@MonarchInit"
+            target="_blank"
```

Comment:
FYI, anytime you use `target="_blank"` you should also add `rel="noopener noreferrer"` for security purposes. cf. https://stackoverflow.com/questions/50709625/link-with-target-blank-and-rel-noopener-noreferrer-still-vulnerable

There's probably a nice way to automate this. I believe I've ensured that any links generated via Markdown have this applied, but I'm not sure if Vue Bootstrap is doing that automatically with `b-dropdown-item`.

Reply:
okay, I've fixed that